parm/config/gefs/config.fcst

```diff
@@ -0,0 +1,366 @@
+
+# Microphysics configuration
+export dnats=0
+export cal_pre=".true."
+export do_sat_adj=".false."
+export random_clds=".true."
+
+case ${imp_physics} in
+    99) # ZhaoCarr
+        export ncld=1
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_zhaocarr${tbf}${tbp}"
+        export nwat=2
+        ;;
+    6)  # WSM6
+        export ncld=2
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_wsm6${tbf}${tbp}"
+        export nwat=6
+        ;;
+    8)  # Thompson
+        export ncld=2
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_thompson_noaero_tke${tbp}"
+        export nwat=6
+
+        export cal_pre=".false."
+        export random_clds=".false."
+        export effr_in=".true."
+        export ltaerosol=".false."
+        export lradar=".false."
+        export ttendlim="-999"
+        export dt_inner=$((DELTIM/2))
+        export sedi_semi=.true.
+        if [[ "${sedi_semi}" == .true. ]]; then export dt_inner=${DELTIM} ; fi
+        export decfl=10
+
+        export hord_mt_nh_nonmono=5
+        export hord_xx_nh_nonmono=5
+        export vtdm4_nh_nonmono=0.02
+        export nord=2
+        export dddmp=0.1
+        export d4_bg=0.12
+        ;;
+    11) # GFDL
+        export ncld=5
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_gfdl${tbf}${tbp}"
+        export nwat=6
+        export dnats=1
+        export cal_pre=".false."
+        export do_sat_adj=".true."
+        export random_clds=".false."
+        export lgfdlmprad=".true."
+        export effr_in=".true."
+        export reiflag=2
+
+        export hord_mt_nh_nonmono=5
+        export hord_xx_nh_nonmono=5
+        export vtdm4_nh_nonmono=0.02
+        export nord=2
+        export d4_bg=0.12
+        export dddmp=0.1
+        ;;
+    *) echo "Unknown microphysics option, ABORT!" ;;
+esac
+
+# Stochastic physics
+export DO_SPPT=${DO_SPPT:-"NO"}
+export DO_SKEB=${DO_SKEB:-"NO"}
+export DO_SHUM=${DO_SHUM:-"NO"}
+export DO_LAND_PERT=${DO_LAND_PERT:-"NO"}
+export DO_CA=${DO_CA:-"YES"}
+
+#coupling settings
+export cplmode="nems_frac"
+if [[ "${FRAC_GRID:-".true."}" == ".false." ]]; then
+  export cplmode="nems_orig"
+fi
+export psm_bc="1"
+
+export min_lakeice="0.15"
+export min_seaice=${min_seaice:-"0.15"}
+export use_cice_alb=${use_cice_alb:-".false."}
+
+export FSICL="0"
+export FSICS="0"
+
+#---------------------------------------------------------------------
+
+# ideflate: netcdf zlib lossless compression (0-9): 0 no compression
+# nbits: netcdf lossy compression level (0-32): 0 lossless
+export ideflate=1
+export nbits=14
+export ishuffle=0
+# compression for RESTART files written by FMS
+export shuffle=1
+export deflate_level=1
+
+#---------------------------------------------------------------------
+# Disable the use of coupler.res; get model start time from model_configure
+export USE_COUPLER_RES="NO"
+
+if [[ "${CDUMP}" =~ "gdas" ]] ; then # GDAS cycle specific parameters
```

Comment:
Can CDUMP=gefs? For the conditions Lines 280-349, it should keep only for GEFS part. Or I will do this.

Reply:
Same as above. We'll strip it down later.